.github/workflows/e2e.yml

```diff
@@ -1,8 +1,8 @@
 name: e2e
-on: [push, pull_request]
+on: [pull_request]
 jobs:
-  e2e-test:
-    name: E2E test
+  e2e-test-cloud:
```

Comment:
Do we want to test with a higher go-version? like 1.15 or 1.16?

Reply:
Yes, I think that we can verify it with other versions of GO. Just add such matrix to the workflow. I have added all versions 1.13 -> 1.17 but we can narrow that matrix to other versions.